<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ALL_look</class>
 <widget class="QMainWindow" name="ALL_look">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="look2_pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="look1_pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="fix1_pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="look3_pushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PORT:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ip1_lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.20.169</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="back_pushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>380</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="port1_spinBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>8888</number>
    </property>
   </widget>
   <widget class="QLabel" name="look1_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(21, 11, 4);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>无信号！</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="look2_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(21, 11, 4);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>无信号！</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="look3_label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(21, 11, 4);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>无信号！</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="port2_spinBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>6666</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PORT:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fix2_pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fix3_pushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>320</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="port3_spinBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>260</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>5555</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PORT:</string>
    </property>
   </widget>
   <zorder>look3_label</zorder>
   <zorder>look3_pushButton</zorder>
   <zorder>look1_label</zorder>
   <zorder>look1_pushButton</zorder>
   <zorder>look2_label</zorder>
   <zorder>look2_pushButton</zorder>
   <zorder>fix1_pushButton</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>ip1_lineEdit</zorder>
   <zorder>back_pushButton</zorder>
   <zorder>port1_spinBox</zorder>
   <zorder>port2_spinBox</zorder>
   <zorder>label_3</zorder>
   <zorder>fix2_pushButton</zorder>
   <zorder>fix3_pushButton</zorder>
   <zorder>port3_spinBox</zorder>
   <zorder>label_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
